### DISCUSSION TITLE
Latest Browser versions & Backspace, Delete Key events in Terminal

### DISCUSSION THREAD

**Discussion by Harish23-github** [sourcelair/xterm.js#4573] [General]
Hi. 
1. You have mentioned xterm will works in latest version browsers like **Specifically the latest versions of Chrome, Edge, Firefox, and Safari.** Can you please elaborate the versions?
2. Backspace and delete key events are restricted in terminal view. We can able to capture the key events, but not reflecting in terminal window. How to enable that in terminal? I have changed **backspaceUnindent** option to true, but still not reflecting in terminal
3. Can we use xterm for prod environment?

**Comment by jerch** [comment]
1. xtermjs was used all over the world with recent browser versions in the last 7 years. So you can expect it to work at least with versions of all major browsers within the last 5 years. More recent versions are only tested on more recent browser versions though.
2. No clue what you mean by "Backspace and delete key events are restricted in terminal view". Care to elaborate?
3. Yes. it prolly wont get any more production-ready as it already is (running on millions of computers wold-wide through different integrations). Whether it is suitable for your specific task, depends a lot on what you need the terminal for. Most installations use it to bridge to a real PTY on some terminal server.

**Reply by Harish23-github** [reply]
Hey Hi @jerch,  thankyou so much for your response. Regarding #point 2 I found solution.
